backpack-compose/tokens.gradle.kts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import net.skyscanner.backpack.tokens.*
+
+tasks {
+
+  val tokensPackage = "net.skyscanner.backpack.compose.tokens"
+  val group = "tokens"
+  val src = project.projectDir.resolve("src/main/kotlin").path
```

Comment:
Is it a conscious decision to move away from /java here like we have in the rest of the project?